util/asm_dev/armv8/p256/Makefile

```diff
@@ -0,0 +1,41 @@
+#------------------------------------------------------------------------------------
+# Copyright (C) Amazon
+#------------------------------------------------------------------------------------
+
+EXE = p256
+
+SRCDIR = ./src
+BUILDDIR = ./build
+
+
+CFLAGS = -I$(SRCDIR) -std=c11 -MMD
+CFLAGS += -Wall -Wextra -Wpedantic
+CFLAGS += -Werror -Wshadow -Wconversion
+CFLAGS += -march=armv8.4-a+sve -g -O2
```

Comment:
Why is this code compiled with `-march=armv8.4-a+sve`?
I do not see SVE intrinsics in the C code below or SVE instructions in the assembly files.
Also there is no need to require armv8.4 for the following code.
Please remove the `-march` flag.